test/e2e/cypress/e2e/clusters_overview.cy.js

```diff
@@ -81,7 +82,7 @@ context('Clusters Overview', () => {
           clusterIdByName(healthyClusterScenario.clusterName),
           healthyClusterScenario.checks
         );
-        cy.setMockRunnerExpectedResult(healthyClusterScenario.result);
+        // wip: set expected results
```

Comment:
I guess you plan here additional rework ?

Reply:
I don't know...
This were some test that were `skipped` since long time ago.
I just wanted to remove the dead code.
If somebody wants to work on them, feel free hehe